src/chapters/fetch-json/data-formats-json.rst

```diff
@@ -0,0 +1,181 @@
+the word "Astronaut".
+
+.. sourcecode:: js
+   :linenos:
+
+   {
+       "hits": 3,
+       "book": [
+           {
+               "title": "An Astronaut's guide to life on Earth",
+                "author": "Chris Hadfield",
+                "ISBN": 9780316253017,
+                "year_published": 2013,
+                "subject": ["Hadfield, Chris", "Astronauts", "Biography"],
+                "available": true
+           },
+           {
+               "title": "Astronaut",
+               "author": "Lucy M. George",
+               "ISBN": 9781609929411,
+               "year_published": 2016,
+               "subject": ["Astronauts", "Juvenille Fiction", "Space stations"],
+               "available": false
+           },
+           {
+               "title": "Astronaut Ellen Ochoa",
+               "author": "Heather E. Schwartz",
+               "ISBN": 9781512434491,
+               "year_published": 2018,
+               "subject": ["Ochoa Ellen", "Women astronauts", "Astronauts", "Biography", "Women scientists", "Hispanic American women"],
+               "available": true
+           }
+       ]
+   }
+
+This time our search term "Astronaut" returned multiple books, and so a collection of book objects was returned in JSON format.
+
+Each book object can be found in the array with the key ``"book"``. Each book contains the keys title, author, ISBN, year_published,
+subject, and available.
+
+When we make a request to an API, the API formats the data we requested into JSON, and then responds to our request with the
+JSON representation of our request.
+
+
+JSON & JavaScript Object Differences
+------------------------------------
+
+JSON is based on how JavaScript objects work. However, there are some key differences between the two.
+
+JSON keys MUST be in double quotes. When defining properties for a JavaScript object double quotes should not be used when
+declaring properties.
+
+JSON
+
+.. sourcecode:: js
+   :linenos:
+
+   {
+       "title": "The Cat in the Hat"
+       "author": "Dr. Seuss"
+   }
+
+JavaScript object
+
+.. sourcecode:: js
+   :linenos:
+
+   let newBook = {
+       title: "The Cat in the Hat",
+       author: "Dr. Seuss"
+   }
+
+To represent a string in JSON you MUST use double quotes. In JavaScript you can use double quotes, or single quotes.
+
+JSON
+
+.. sourcecode:: js
+   :linenos:
+
+   {
+       "title": "The Last Astronaut",
+       "author": "David Wellington"
+   }
+
+JavaScript object
+
+.. sourcecode:: js
+   :linenos:
+
+   let anotherBook = {
+       title: 'The Last Astronaut',
+       author: 'David Wellington'
+   }
+
+.. note::
+
+   JSON is based on JavaScript objects, but there are key differences. JSON syntax is a little more strict than JavaScript
+   object syntax. Luckily JavaScript has built in tools to turn JSON into a JavaScript object with relative ease. These tools
+   will be introduced later in the course.
```

Comment:
@pdmxdd What tools? Maybe JSON.parse() and JSON.stringify?

Reply:
took it out